Classes/Core/Testbase.php

```diff
@@ -966,14 +1018,15 @@ protected function copyRecursive($from, $to)
 
     /**
      * Get Path to vendor dir
-     * Since we are installed in vendor dir, we can safely assume the path of the vendor
-     * directory relative to this file
+     * Since we are surly installed in the vendor dir, we can reuse the composer runtime information to get the
+     * correct installation path of the testing-framework. With that we can calculate the package folder precisely,
+     * avoiding invalid path determination when symlinks has been invited to the party.
      *
      * @return string Absolute path to vendor dir, without trailing slash
      */
     public function getPackagesPath(): string
     {
-        return rtrim(strtr(dirname(__DIR__, 4), '\\', '/'), '/');
+        return $this->composerPackageManager->getVendorPath();
```

Comment:
like this this method is superfluous, isn't it?  
`$this->composerPackageManager->getVendorPath()` is more descriptive and clear anyway, imo.

Reply:
We can deprecate it (later) or rename it, but IMO I would keep it because of compat reasons (see UnitBootstrap). That way, it works without being breaking.  For internal usage, we can directly use it from the composerPackageManager.

My overall goal was to keep needed changes for consumer as low as possible.

Reply:
ok, then just leave it for the moment.

Reply:
Would like to post-pone this to a dedicated change afterwards. And deciding if main only, or 7 to .. and if deprecation should be done or simply renaming.

Will discuss this with lolli. But postponing for now.